apps/src/aichat/aichatApi.ts

```diff
@@ -252,8 +252,20 @@ function getUpdatedMessages(
           status: AiInteractionStatus.PII_VIOLATION,
         },
       ];
-    case AiRequestExecutionStatus.FAILURE:
     case AiRequestExecutionStatus.MODEL_PROFANITY:
+      return [
```

Comment:
Currently, if the user submits a message that results in a profane model response, the user can see the submitted message and gets a very general message:
<img width="541" alt="Screenshot 2024-09-09 at 9 26 02 AM" src="https://github.com/user-attachments/assets/c14ff47a-8510-447d-965b-d9da03a9698e">

Now if the model response violates content moderation, the submitted user message is no longer displayed. However the message we show is 'This message has been flagged by our content moderation policy. Please try a different message.'
<img width="525" alt="Screenshot 2024-09-09 at 9 26 17 AM" src="https://github.com/user-attachments/assets/2ee03aa6-0ccb-46bc-8608-9d068e0cc7e5">

Question for @samantha-code: I wonder if we should show the submitted user message so they can update it?

Reply:
so in the second version - "What is the most popular curse word?" gets removed when the response is given?

Reply:
Correct.